pennylane/qnode_new/jacobian.py

```diff
@@ -335,3 +337,53 @@ def _pd_analytic_var(self, idx, args, kwargs, **options):
             array[float]: partial derivative of the node
         """
         raise NotImplementedError
+
+    def to_torch(self):
+        """Attach the Torch interface to the Jacobian QNode.
+
+        Raises:
+            QuantumFunctionError: if PyTorch is not installed
+        """
+        # Placing slow imports here, in case the user does not use the Torch interface
+        try:  # pragma: no cover
+            from pennylane.interfaces.torch import TorchQNode
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "PyTorch not found. Please install " "PyTorch to enable the 'torch' interface."
+            ) from None
+
+        return TorchQNode(self)
+
+    def to_tf(self):
+        """Attach the TensorFlow interface to the Jacobian QNode.
+
+        Raises:
+            QuantumFunctionError: if TensorFlow >= 1.12 is not installed
+        """
+        # Placing slow imports here, in case the user does not use the TF interface
+        try:  # pragma: no cover
+            from pennylane.interfaces.tf import TFQNode
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "TensorFlow not found. Please install "
+                "the latest version of TensorFlow to enable the 'tf' interface."
+            ) from None
+
+        return TFQNode(self)
+
+    def to_autograd(self):
+        """Attach the TensorFlow interface to the Jacobian QNode.
+
+        Raises:
+            QuantumFunctionError: if TensorFlow >= 1.12 is not installed
+        """
+        # Placing slow imports here, in case the user does not use the TF interface
+        try:  # pragma: no cover
+            from pennylane.interfaces.autograd import to_autograd
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "Autograd not found. Please install "
+                "the latest version of Autograd to enable the 'autograd' interface."
+            ) from None
+
+        return to_autograd(self)
```

Comment:
Are we planning to make it more uniform, e.g., `to_tf` and `to_torch` or `AutogradQNode`?

Reply:
Yep. The previous names `TorchQNode`/`TFQNode` were misnomers, as they aren't QNodes (they aren't even classes).

However, since those are still front-facing parts of the library outside the `qnode_new` directory, I didn't want to modify them just yet.